frontend/awx/access/users/UserPage/UserTokens.tsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { usePageNavigate, PageNotImplemented } from '../../../../../framework';
+import { AwxRoute } from '../../../main/AwxRoutes';
+import { useAwxActiveUser } from '../../../common/useAwxActiveUser';
+
+export function UserTokens() {
+  const params = useParams<{ id: string }>();
+  const [showTokens, setShowTokens] = useState(true);
+  const { activeAwxUser } = useAwxActiveUser();
+  const pageNavigate = usePageNavigate();
+
+  useEffect(() => {
+    if (activeAwxUser?.id === undefined || activeAwxUser?.id.toString() !== params.id) {
+      setShowTokens(false);
+      pageNavigate(AwxRoute.Users);
```

Comment:
It might be better to remain in the context of the already selected user. Can we navigate instead to `AwxRoute.UserDetails` passing id as a param?

Reply:
Good point, I will make that change.